hack/kustomize-sub.sh

```diff
@@ -17,5 +17,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-root=$(dirname "${BASH_SOURCE[0]}")
-"$root/tools/bin/kustomize" build "$1" | "$root/tools/bin/envsubst"
\ No newline at end of file
+root=$(dirname "${BASH_SOURCE[0]}")/..
+script_invocation_dir="$(pwd)"
+kustomize="${root}/hack/tools/bin/kustomize"
+envsubst="${root}/hack/tools/bin/envsubst"
+
+# go to root and install kustomize, envsubst if missing
+cd ${root} && make "${kustomize##*/}"; make ${envsubst##*/}
```

Comment:
Same for here

Reply:
Addressed via commit dc6d1f602487940b00cbd53b59f014d7fc4bf089